src/pybind/ceph_volume_client.py

```diff
@@ -660,8 +664,14 @@ def deauthorize(self, volume_path, auth_id):
         pool_name = self._get_ancestor_xattr(path, "ceph.dir.layout.pool")
         namespace = self.fs.getxattr(path, "ceph.dir.layout.pool_namespace")
 
-        want_mds_cap = 'allow rw path={0}'.format(path)
-        want_osd_cap = 'allow rw pool={0} namespace={1}'.format(pool_name, namespace)
+        # The auth_id might have read-only or read-write mount access for the
```

Comment:
The ambiguity would be over whether when there is an 'r' rule and a 'rw' rule, should deauthorize(...'rw') remove both rules, or should it leave the 'r' rule in place.  I would suggest the latter, so that this is behaving the same way as authorize().

It would be great to have some tests for these kinds of cases, as a way of demonstrating the behaviour